src/store/modules/app/executeRequest.js

```diff
@@ -41,30 +39,32 @@ export const executeRequest = async ({ rootGetters, dispatch, rootState }, { req
       call = methodFunc(...request.args)
     }
 
-    const { ledgerBridgeConnected, ledgerBridgeTransportConnected } = rootState.app
-    if (account?.type.includes('ledger')) {
-      if (!ledgerBridgeConnected) {
-        dispatch('startBridgeListener', {
-          onConnect: () => {
-            resolve(call)
-          }
-        }).then(() => {
-          dispatch('openUSBBridgeWindow')
-        })
-      } else if (!ledgerBridgeTransportConnected) {
-        dispatch('startBridgeListener', {
-          onTransportConnect: () => {
-            resolve(call)
-          }
-        }).then(() => {
-          dispatch('openUSBBridgeWindow')
-        })
-      } else {
-        resolve(call)
-      }
-    } else {
-      resolve(call)
-    }
+    resolve(call)
+
+    // const { ledgerBridgeConnected, ledgerBridgeTransportConnected } = rootState.app
```

Comment:
What about injection though? Does ledger integration work with injection after this change?

Reply:
right it should work, the connection request should be in the popup and after that we can call the client directly